src/components/Questions/Header.vue

```diff
@@ -54,6 +92,53 @@ export default defineComponent({
         "text-white text-sm bp-500:text-md lg:text-lg xl:text-xl font-bold",
     }));
 
+    const timerButtonClass = computed(() => {
+      let buttonClass;
+      if (state.timeRemaining >= props.warningTimeLimit * 60) {
+        buttonClass = state.timerButtonNormal;
+      } else {
+        buttonClass = state.timerButtonAlert;
+      }
+      return buttonClass;
+    })
+
+    function appendZeroForSingleDigits(digit: Number) {
```

Comment:
Maybe this is a better name?
```suggestion
    function padWithZeroes(digit: Number) {
```